source-map-support.js

```diff
@@ -555,6 +555,19 @@ exports.install = function(options) {
     }
   }
 
+  // Use dynamicRequire to avoid including in browser bundles
+  var Module = dynamicRequire(module, 'module');
+
+  // Redirect subsequent imports of "source-map-support"
+  // to this package
+  const originalResolveFilename = Module._resolveFilename;
+  Module._resolveFilename = function resolveFilenameProxy(request, parent, isMain, options) {
+    if (request === 'source-map-support') {
+      request = '@cspotcode/source-map-support';
```

Comment:
This may break under certain environments if a library declares a dependency on `source-map-support` but not on `@cspotcode/source-map-support`.  For example, yarn PnP will not allow a require to resolve if it is not declared via `package.json` `dependencies`, `devDependencies`, or `peerDependencies`.  We'll need to return the fully-resolved path to `@cspotcode/source-map-support`.